DevelopedCalculator.js

```diff
@@ -30,6 +30,8 @@ class DevelopedCalculator extends Calculator {
     onOperatorButtonClick(event) {
         // calling parent class's onOperatorButtonClick method
         super.onOperatorButtonClick(event);
+        const currentResultAndclickedOperButton = this.currentResult + event.target.dataset.operator;
+        this.currentResult = currentResultAndclickedOperButton;
         // pushing the currentResult to the history array 
         this.history.push(this.currentResult);
```

Comment:
can you tell the reason behind pushing the `currentResult` here?

Reply:
I am pushing the the current result and clicked operator now to update the history element